operator/apis/mlops/v1alpha1/experiment_types.go

```diff
@@ -154,7 +154,7 @@ func (es *ExperimentStatus) IsConditionReady(t apis.ConditionType) bool {
 func (es *ExperimentStatus) SetCondition(conditionType apis.ConditionType, condition *apis.Condition) {
 	switch {
 	case condition == nil:
-		experimentConditionSet.Manage(es).MarkUnknown(conditionType, "", "")
+		return
```

Comment:
could you look for all/similar occurrences for this `SetCondition` helper and follow the same pattern you have suggested here?

Reply:
I have updated all the cases to have similar pattern  for `SetCondition`